src/views/settings/Security/Auth.vue

```diff
@@ -57,7 +55,6 @@ export default {
               change: ([val], updateForm) => {
                 if (val) {
                   updateForm({ SECURITY_LOGIN_CHALLENGE_ENABLED: false })
-                  updateForm({ SECURITY_LOGIN_CAPTCHA_ENABLED: false })
                 }
               }
             }
```

Comment:
The main issue I noticed is that there are a few typos throughout the code; especially in some of the variable names such as `sec` rather than `security`. There appear to be slight discrepancies between lines 24 and 25 where it seems an additional space was added when adding a comma. Other than those minor formatting changes, overall the code follows good coding standards with clear function definition. No major bugs or improvements have been suggested.